src/ensembl/tests/fixtures/entity-viewer/product.ts

```diff
@@ -40,7 +40,10 @@ export const createProduct = (fragment: Partial<Product> = {}): Product => {
     length: length,
     protein_domains: createProteinDomains(length),
     external_references: times(2, () => createExternalReference()),
-    ...fragment
+    ...fragment,
```

Comment:
fragment should be the last, to be able to override any of the above properties

Reply:
Done